consumer-backend/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "consumer.js",
   "scripts": {
     "lint": "eslint .",
-    "openshift": "npx nodeshift --dockerImage=registry.access.redhat.com/ubi8/nodejs-14",
+    "openshift": "npx nodeshift --dockerImage=registry.access.redhat.com/ubi8/nodejs-12",
```

Comment:
any reason for the ndoe version downgrade?

Reply:
I probably should have mentioned : https://bugzilla.redhat.com/show_bug.cgi?id=1950018.  installing from Git seems broken in the 14.x container.

Reply:
Seems like we are also missing openssl headers in the 14.x containers, so for node-rdkafka we should leave this as nodejs-12 for now.